uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/service/UniProtEntryService.java

```diff
@@ -130,9 +127,9 @@ private SolrRequest createSolrRequest(SearchRequestDTO request, boolean includeF
         return requestBuilder.build();
     }
 
-    private Sort getUniProtSort(String sortStr, boolean hasScore) {
+    private Sort getUniProtSort(String sortStr) {
         if (Strings.isNullOrEmpty(sortStr)) {
-            return UniProtSortUtil.createDefaultSort(hasScore);
+            return UniProtSortUtil.createDefaultSort(true);
```

Comment:
It will always be true? is there any other place that it might be false?  if that is the case, we can remove this parameter. I am not sure....